src/openrct2/interface/Viewport.cpp

```diff
@@ -202,6 +192,28 @@ void viewport_create(
     viewport->viewPos = *centreLoc;
 }
 
+void viewport_remove(rct_viewport* viewport)
```

Comment:
TitleCase all the new functions

Reply:
I want to refactor all this code into a namespace after this PR, its a big file.

Reply:
Is there a reason why this cannot be put in the member function Viewport::Remove()?

Reply:
Rather a habbit but I guess its trivial to add, I'll see to do that.

Reply:
There is a reason you wouldn't do this... you typically tell the manager to remove a child, not the child to remove itself. So it would be more acceptable to do ViewportManager::Remove(viewport). Otherwise you typically have to set a flag on the viewport to self-delete at the end of the update or something since you wouldn't want to `delete this` within a member.